mod/admin.php

```diff
@@ -171,17 +171,30 @@ function admin_content(App $a)
 	// array(url, name, extra css classes)
 	// not part of $aside to make the template more adjustable
 	$aside_sub = [
-		'site'         => ["admin/site/"        , L10n::t("Site")                 , "site"],
-		'users'        => ["admin/users/"       , L10n::t("Users")                , "users"],
-		'addons'       => ["admin/addons/"      , L10n::t("Addons")               , "addons"],
-		'themes'       => ["admin/themes/"      , L10n::t("Themes")               , "themes"],
-		'features'     => ["admin/features/"    , L10n::t("Additional features")  , "features"],
-		'dbsync'       => ["admin/dbsync/"      , L10n::t('DB updates')           , "dbsync"],
-		'queue'        => ["admin/queue/"       , L10n::t('Inspect Queue')        , "queue"],
-		'contactblock' => ["admin/contactblock/", L10n::t('Contact Blocklist')    , "contactblock"],
-		'blocklist'    => ["admin/blocklist/"   , L10n::t('Server Blocklist')     , "blocklist"],
-		'federation'   => ["admin/federation/"  , L10n::t('Federation Statistics'), "federation"],
-		'deleteitem'   => ["admin/deleteitem/"  , L10n::t('Delete Item')          , 'deleteitem'],
+		'information' => [ L10n::t('Information'), [
+			"overview" => ["admin/", L10n::t("Overview"), "overview" ],
+			'federation'   => ["admin/federation/"  , L10n::t('Federation Statistics'), "federation"] ]],
```

Comment:
Next time, please decide on a single quote style for the whole function/file.

Reply:
Copy and paste oversight under loud construction work nearby ;-)

Reply:
Sorry, I missed this by reviewing it.